pulser/sequence.py

```diff
@@ -643,9 +643,11 @@ def deserialize(obj, **kwargs):
         return json.loads(obj, cls=PulserDecoder, **kwargs)
 
     @_screen
-    def draw(self):
+    def draw(self, draw_phase_area=False):
```

Comment:
Can you add info on the `draw_phase_area` to this docstring?

Reply:
Added it now, thanks, I forgot to do it earlier.